js/ui/toolbar.js

```diff
@@ -55,7 +55,7 @@ var Toolbar = ToolbarBase.inherit({
             * @name dxToolbarOptions_submenuType
             * @publicName submenuType
             * @type string
-            * @default 'dxDropDownMenu'
+            * @default 'dropDownMenu'
```

Comment:
@DokaRus 

Do we need doc comments for this hidden option?

Reply:
We decided do not remove it for now